src/common/models/Projection.ts

```diff
@@ -491,6 +514,41 @@ export class Projection {
     );
   }
 
+  private calcCaseDensityByCases(): Array<number | null> {
+    const caseDensityByCases = [];
+    const totalPopulation = this.totalPopulation;
+    for (let i = 0; i < this.dates.length; i++) {
+      const weeklyAverage = this.getWeeklyAverageForDay(
+        this.dailyPositiveTests,
+        i,
+      );
+      if (weeklyAverage === null) {
+        caseDensityByCases.push(null);
+      } else {
+        const val = weeklyAverage / (totalPopulation / 100000);
+        caseDensityByCases.push(val);
+      }
+    }
+    return caseDensityByCases;
+  }
+
+  private calcCaseDensityByDeaths(): Array<number | null> {
+    const caseDensityByDeaths = [];
+    const totalPopulation = this.totalPopulation;
+    for (let i = 0; i < this.dates.length; i++) {
+      const weeklyAverage = this.getWeeklyAverageForDay(this.dailyDeaths, i);
+      if (weeklyAverage === null) {
+        caseDensityByDeaths.push(null);
+      } else {
+        const estimatedCaseFatalityRatio = 0.01; // 1% of cases expected to turn into deaths.
```

Comment:
nit: Consider creating a constant at the top of the file for this ratio so it's easier to find if we have to change it (we might be using this number for other calculations, I'm going to check). I'm not sure, but I _think_ they use "mortality" no?